pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -930,6 +939,7 @@ func PodEnv(i *infinispanv1.Infinispan, systemEnv *[]corev1.EnvVar) []corev1.Env
 		{Name: "JAVA_OPTIONS", Value: i.GetJavaOptions()},
 		{Name: "EXTRA_JAVA_OPTIONS", Value: i.Spec.Container.ExtraJvmOpts},
 		{Name: "DEFAULT_IMAGE", Value: consts.DefaultImageName},
+		{Name: "ADMIN_IDENTITIES_PATH", Value: consts.ServerAdminIdentitiesPath},
```

Comment:
Do we need to add admin secret reflection mechanism. I think we need to give clients the ability to change the admin password for some security reason.

Reply:
It is possible for the user to create the admin credentials by creating the admin secret before installing/restarting the operator.

Reply:
It would now be necessary for the user to create the `<infinispan-cr-name>-generated-admin-secret` before creating the Infinispan CR if they require custom credentials. I can add the HASH and restart logic if we think this is required.

Reply:
I think that ability to change credentials is enough for user

Reply:
@ryanemerson are you plan to add ADMIN_IDENTITIES_HASH for this feature?